email_automation/welcome-template.txt

```diff
@@ -0,0 +1,34 @@
+<html>
+<head></head>
+<body>
+<h1>Welcome to OpenPATH!</h1>
+<p> Thank you for joining the OpenPATH project.<br>
+<br>
+ <strong>Your admin dashboard can be found at:</strong> <br>
+ <br>
+    <a href='https://<pool_name>-openpath.nrel.gov/admin/'>
+    https://<pool_name>-openpath.nrel.gov/admin/</a>.<br>
+    <br>
+    An account has been created for you with AWS Cognito. Please log in and change your 
+    temporary password:<br>
+    <br>
+    Username: {username}
+    Temporary Password: {####}
+    <br>
+    <br>
+        The login to the admin dashboard uses 2-factor authentication. You will have to install an authenticator app (e.g. Microsoft Authenticator, Google Authenticator) to receive One Time Passcodes (OTP) every time you want to log in. This is a requirement from NREL cyber to ensure that the data access to individual user information is secure.<br>
+      <br>
+    You can download trip and user tables as csv and view aggregate origin-destination points in the maps.<br>
+    <br>
+        Since you indicated that you want to map the data to infrastructure updates, your configuration includes trip start/end in the trip table. Let us know if you would like to exclude those for greater anonymity.
```

Comment:
Ditto. This should be based on whether there are `"data_trips_columns_exclude"` columns or not.
If we do exclude the trip start/end, then they are already excluded, and this text should be changed.

Reply:
> Updated this. See example of how it works [here](https://github.com/e-mission/e-mission-docs/issues/1008#issuecomment-1830736893)

see above